Driver_FET.dcm

```diff
@@ -690,6 +702,12 @@ K Gate Driver
 F https://www.infineon.com/dgdl/Infineon-IRS2890DS-DS-v01_00-EN.pdf?fileId=5546d4625a888733015aad6fbc8a4bf4
 $ENDCMP
 #
+$CMP L6491
+D 600V Half Bridge Gate Driver with OCP and Integrated Bootstrap Diode, 4A drive current, DSO-14
+K Gate Driver MOSFET IGBT
+F https://www.st.com/content/ccc/resource/technical/document/datasheet/39/4a/2e/5f/bc/cf/42/20/DM00088170.pdf/files/DM00088170.pdf/jcr:content/translations/en.DM00088170.pdf
```

Comment:
Please change to `https://www.st.com/resource/en/datasheet/l6491.pdf`.

Reply:
Done